.github/workflows/after-release.yml

```diff
@@ -16,7 +16,7 @@ jobs:
 
   update-changelog:
     runs-on: [ ubuntu-latest ]
-    if: github.event_name == 'release'
+    if: github.event_name == 'release' && github.repository == 'App-vNext/Polly'
```

Comment:
These could all be changed to `github.event.repository.fork == false` - then if the repo were ever moved/renamed, the semantics would remain the same without a need to update all the naming.

Reply:
That's a very good point, thank you for the feedback! :smile: